tests/db/conftest.py

```diff
@@ -115,7 +120,7 @@ def inputs(user_vk):
     for block in range(4):
         transactions = [
             Transaction.create(
-                [b.me], [user_vk], payload={'i': i}).sign([b.me_private])
+                [b.me], [user_vk], metadata={'i': i}).sign([b.me_private])
```

Comment:
> And metadata also generates a unique uuid, right?

Yes, though if metadata is `None`, no uuid is generate, making the Transaction not necessarily unique. With the requirement for always having an asset instantiated now, we can be sure of a Transaction's uniqueness.